x-pack/packages/kbn-data-forge/src/data_sources/fake_hosts/index.ts

```diff
@@ -91,6 +91,7 @@ export const generateEvent: GeneratorFunction = (config, schedule, index, timest
       '@timestamp': timestamp.toISOString(),
       host: {
         name: `host-${index}`,
+        mac: ['00-00-5E-00-53-23', '00-00-5E-00-53-24'],
```

Comment:
@simianhacker I've added `host.mac` to all the docs in fake_hosts, please let me know if you see an issue.